### PR TITLE
Added new extension

### PR DESCRIPTION
**Pull request by jtanadi**
checkParallelTool

### CODE REVIEW

File: _data/checkParallelTool.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+description: Draws a line between on-curve points and BCPs and checks if they're parallel.
+extensionName: Check Parallel Tool
+repository: https://github.com/jtanadi/CheckParallelTool
+extensionPath: CheckParallelTool.roboFontExt
+developer: Jesen Tanadi
+developerURL: http://http://jesentanadi.com
```
Comment (typemytype) [inline _data/checkParallelTool.yml]: http:// is there twice

**Comment by jtanadi** [comment]
Can I add one later, or is that annoying?

--  
Jesen Tanadi

**Comment by jtanadi** [comment]
Oops, sorry. Will fix in a little bit.  

--  
Jesen Tanadi

**Comment by typemytype** [comment]
you can add icon later, it could be 512x512px image of your toolbar icon...

the url must be fixed ;)

**Comment by jtanadi** [comment]
Thanks Frederik!

**Pull request merged**